### PR TITLE
Issue 144 update text swagger

### PR DESCRIPTION
**Pull request by TheAyeshaSiddiqa**
Added header Texts for API methods.
image

### CODE REVIEW

File: .flake8 [hunk 1]
```diff
@@ -1,3 +1,3 @@
 [flake8]
-max-line-length = 88
+max-line-length = 150
```
Comment (BarisSari) [inline .flake8:2]: I think this is too much, @TheAyeshaSiddiqa. Can you make it 120 maximum and update the texts?

File: .envs/.local/.sls [hunk 2]
```diff
@@ -0,0 +1,7 @@
+MINIO_ACCESS_KEY=enrolments
+MINIO_SECRET_KEY=wJalrXUtnFEMIK7MDENGbPxRfiCYEXAMPLEKEY
+S3_ACCESS_KEY_ID=enrolments
+S3_SECRET_ACCESS_KEY=wJalrXUtnFEMIK7MDENGbPxRfiCYEXAMPLEKEY
+S3_ENDPOINT_URL=http://minio:9000
+CALLBACK_BUCKET=put-callbacks-here
+ENROLMENT_BUCKET=put-enrolments-here
```
Comment (BarisSari) [inline .envs/.local/.sls:7]: I think we don't need them anymore. Did you check this PR: https://github.com/ACWIC/employer-admin/pull/27

File: .envs/.local/.minio [hunk 3]
```diff
@@ -0,0 +1,2 @@
+MINIO_ACCESS_KEY=enrolments
+MINIO_SECRET_KEY=wJalrXUtnFEMIK7MDENGbPxRfiCYEXAMPLEKEY
```
Comment (BarisSari) [inline .envs/.local/.minio:2]: I think we don't need them anymore. Did you check this PR: https://github.com/ACWIC/employer-admin/pull/27

File: app/api/enrolments.py [hunk 4]
```diff
@@ -33,6 +37,19 @@ def create_enrolment(inputs: NewEnrolmentRequest):
...
+    <p>
+    Aged Care Provider (employer) can view enrolment they posted.\n
+    He will get,
+    <ol>
+      <li><b>enrolment_id</b> identifier for enrolment</li>
+      <li><b>shared_secret</b> to recieve callback</li>
+      <li><b>internal_reference</b> employer_reference</li>
+      <li><b>created</b> date and time when the enrolment was created</li>
+    </ol>
+    <b>internal_reference</b> a unique, non-empty string for employer_reference.\n
```
Comment (BarisSari) [inline app/api/enrolments.py:50]: I think you can write this to above (`<li><b>internal_reference</b>` this line)

File: app/api/enrolments.py [hunk 5]
```diff
@@ -71,7 +103,19 @@ def get_callbacks_list_for_enrolment(
 @router.get("/enrolments/{enrolment_id}/journal/{callback_id}")
 def get_callback_for_enrolment(enrolment_id: str, callback_id):
     """
-    Returns callback details for an callback of an enrolment
+    <p>
+    Aged Care Provider (employer) can view callback details for an enrolment\n
+    He will get,
+    <ol>
+      <li><b>callback_id</b> identifier for enrolment</li>
+      <li><b>enrolment_id</b> identifier for callback</li>
```
Comment (BarisSari) [inline app/api/enrolments.py:111]: These two explanations should be switched.

File: app/config.py [hunk 6]
```diff
@@ -41,6 +41,7 @@ def s3_configuration(self) -> dict:
         }
 
     class Config:
+        env_file = ".envs/.local/.sls"
```
Comment (BarisSari) [inline app/config.py:44]: Remove this line. (I shared PR's link above)

File: local.yml [hunk 7]
```diff
@@ -20,14 +20,15 @@ services:
     ports:
       - "8082:8080"
     env_file:
-      - ./.env
+        - ./.envs/.local/.minio
+        - ./.envs/.local/.env
```
Comment (BarisSari) [inline local.yml:24]: Remove these lines (Check comment's above)

File: app/api/enrolments.py [hunk 8]
```diff
@@ -71,7 +103,19 @@ def get_callbacks_list_for_enrolment(
...
+      <li><b>shared_secret</b> for verification</li>
+      <li><b>received</b> date and time when the callback was received</li>
+      <li><b>tp_sequence</b> sequence number from training provider source-system.
+      These are used for sorting in the order of sender’s intent.</li>
+      <li><b>payload</b> required</li>
```
Comment (BarisSari) [inline app/api/enrolments.py:116]: Callback's structure has changed. There is no tp_sequence anymore. Can you check it from the `employer-callback` repository?

**Review by BarisSari — CHANGES_REQUESTED** [review]
I asked for some changes, @TheAyeshaSiddiqa. Actually, I have one more general question. You used HTML for docstrings which are not common. Did you try to style using markdown?

**Comment by TheAyeshaSiddiqa** [comment]
> I asked for some changes, @TheAyeshaSiddiqa. Actually, I have one more general question. You used HTML for docstrings which are not common. Did you try to style using markdown or rst?

I saw it somewhere and it seemed like a good idea, but yeah sure I can change it to markdowns.
And the PR you're referring hasn't been merged yet.

File: app/api/enrolments.py (hunk 8 above)
Comment (TheAyeshaSiddiqa) [reply app/api/enrolments.py:116]: Checked it. I'll open a new PR for it. As it will change 3 usecases.

**Pull request merged**